client/src/components/GoogleAuth.js

```diff
@@ -20,7 +26,8 @@ class GoogleAuth extends React.Component {
   }
   onAuthChange = isSignedIn => {
     if (isSignedIn) {
-      this.props.signIn(this.auth.currentUser.get().getId());
+      this.props.fetchProfile({id: this.auth.currentUser.get().getId(), name: this.auth.currentUser.get().getBasicProfile().getName() , email: this.auth.currentUser.get().getBasicProfile().getEmail()});
```

Comment:
You should probably wrap these lines something like this:
this.props.fetchProfile({
  id: this.auth.currentUser.get().getId(),
  name........
});